spec/models/person_spec.rb

```diff
@@ -35,24 +35,24 @@
 
   context '#skill_list' do
     it 'show array of skills' do
-      person = Person.create(skills: 'Ruby,Ruby on Rails')
+      person = create(:person, skills: 'Ruby,Ruby on Rails')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.